localization/english/event_localization/wc_being_events_l_english.yml

```
@@ -6,4 +6,14 @@
  WCBEI_10_DESC_child:0 "The adults didn't say much about what that means. Will I never see [lost_character.GetHerHim] again?"
  WCBEI_10_DESC_fate:0 "What a shame! It is difficult to imagine a fate worse than this."
  WCBEI_10_OPT_A:0 "A part of [lost_character.GetHerHim] is still alive."
- WCBEI_10_OPT_B:0 "This is not my [ROOT.Char.Custom2('RelationToMeShort', SCOPE.sC('lost_character'))] anymore."
\ No newline at end of file
+ WCBEI_10_OPT_B:0 "This is not my [ROOT.Char.Custom2('RelationToMeShort', SCOPE.sC('lost_character'))] anymore."
+ 
+ WCBEI_100_TITLE:0 "Twisting Nether"
+ WCBEI_100_DESC:0 "I've lost my body in the mortal planes but my soul has found its way to the Twisting Nether.\n\nThousands of twisted demonic souls, merging into a single stream, chasing grains of cosmic energy. Only the strongest may survive, the rest meet oblivion.\n\nI can join the stream, feeding on magic and regain my mortal shell; otherwise, give up and dive into the Great Dark Beyond."
```

I know TheGroxEmpire suggested this one, but I personally think it should be reverted. The semicolon isn't necessary, and the `or` makes it clearer that it is a choice between the two options.
```suggestion
 WCBEI_100_DESC:0 "I've lost my body in the mortal planes but my soul has found its way to the Twisting Nether.\n\nThousands of twisted demonic souls, merging into a single stream, chasing grains of cosmic energy. Only the strongest may survive, the rest meet oblivion.\n\nI can join the stream, feeding on magic and regain my mortal shell or give up and dive into the Great Dark Beyond."
```